src/main/kotlin/no/nav/familie/klage/kabal/KabalService.kt

```diff
@@ -34,8 +37,8 @@ class KabalService(
             kildeReferanse = behandling.eksternBehandlingId.toString(),
             innsynUrl = lagInnsynUrl(fagsak, behandling.påklagetVedtak),
             hjemler = vurdering.hjemmel?.let { listOf(it.kabalHjemmel) } ?: emptyList(),
-            forrigeBehandlendeEnhet = behandling.behandlendeEnhet,
-            tilknyttedeJournalposter = listOf(), // TODO: klagebrev kan puttes på automatisk, vedtaksbrev fra EF-sak må hentes fra iverksett, klage må velges ved ferdigstilling eller ved journalføring av klage
+            forrigeBehandlendeEnhet = saksbehandlersEnhet,
```

Comment:
Burde vi bytte navn på "forrigeBehandlendeEnhet" enhet etterhvert (ikke i denbne pr'en, men ...)?

Reply:
Bør isåfall gjøres av Kabal, men det heter det fremdeles der...